packages/react/src/search-input/docs/overview.mdx

```diff
@@ -21,6 +23,7 @@ By default, the `SearchInput` component does not expand to fill the available sp
 
 - use to filter results on a page
 - provide a meaningful label – for example, Search components
+- wrap in an appropriately labelled `role="search"` landmark
```

Comment:
Should we be more specific around not overriding the `form`'s role?

Reply:
It seems you can do that anyway, should it be the only thing https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/search_role

Reply:
I was reading this [Adrian Roselli article about not overriding the default form role](https://adrianroselli.com/2015/08/where-to-put-your-search-role.html). I assumed you were doing the same thing. What do you think about this article, should we be following the recommendation?

Reply:
Seems it's a non-issue. The [penultimate update](https://adrianroselli.com/2015/08/where-to-put-your-search-role.html#changed) states:

> As of this morning it appears there is agreement that the prohibition against role="search" on <form> elements has been lifted. It is discussed in the following bug reports: